proto-public/go.mod

```diff
@@ -4,18 +4,18 @@ go 1.19
 
 require (
 	github.com/stretchr/testify v1.5.1
-	google.golang.org/grpc v1.37.1
-	google.golang.org/protobuf v1.27.1
+	google.golang.org/grpc v1.56.3
+	google.golang.org/protobuf v1.30.0
```

Comment:
Note that this version was _significantly_ out of date compared w/ the root `go.mod`.